### PR TITLE
Added a GPLV2 license. Also, added an option in the script for listin…

### PR DESCRIPTION
**Pull request by rpchandran**
…g the existing profiles.

### CODE REVIEW

File: aaws.sh [hunk 1]
```diff
@@ -41,6 +41,15 @@ function aaws()
...
   fi
 
+  if [ "-l" == "$@" ]; then
+    printf "Available Profiles:"
+    for e in ${AAWS_AUTOCOMPLETE[@]}; do
+      printf " $e"
+    done
+    printf "\n"
+    return 0
+  fi
```
Comment (luk3thomas) [inline aaws.sh]:
```suggestion
  if [ "-l" == "$@" ]; then
    printf "Available Profiles:\n"
    for e in ${AAWS_AUTOCOMPLETE[@]}; do
      printf "  $e\n"
    done
    return 0
  fi
```

I have a long list of profiles and it is nicer (IMO 😄 ) to see them listed vertically that going out to the side

Looks something like this now

```
$ aaws -l
Available Profiles:
  a
  b
  c
```

**Review by luk3thomas — CHANGES_REQUESTED** [review]
LGTM, thanks for adding the license.

Can you add `-l` to the help?

```
    echo "usages: aaws [options] [profile]"
    echo ""
    echo "  aaws       - Clear the current profile"
    echo "  aaws -h    - Display help text"
    echo "  aaws prod  - Set your AWS_PROFILE to prod"
```

File: aaws.sh (hunk 1 above)
Comment (rpchandran) [reply aaws.sh]: I added help, changed to vertical and removed the "Available Profiles:" text because I think it is cleaner with vertical listing.

Comment (rpchandran) [reply aaws.sh]: I pushed the changes to the forked branch. Not sure what is next for me.

**Review by luk3thomas — APPROVED** [review]

**Pull request merged**